.gitignore

```diff
@@ -25,7 +25,6 @@ buildNumber.properties
 .classpath
 
 # End of https://www.toptal.com/developers/gitignore/api/maven
-=======
```

Comment:
I think this slipped in during a git merge conflict resolution.

Reply:
why ignore classpath?
i use eclipse, and classpath is needed for it to find the sources and compile and build.
presumably other people use eclipse, too...